app/models/maintenance_tasks/task.rb

```diff
@@ -21,6 +21,9 @@ class NotFoundError < NameError; end
     # @api private
     class_attribute :collection_builder_strategy, default: NullCollectionBuilder.new
 
+    # As defined by JobIteration::EnumeratorBuilder#build_active_record_enumerator_on_records as `columns`
```

Comment:
Can you improve this documentation telling what this does instead of linking to method in another library? For the sake of this library, this method is only to configure which column will be used in the Active Record enumerator. Let's explain this and then, we can link to the other library, but if we are going to do that, we should really link to the library documentation, not just mention the reference of the method.